tshirts.spec.js

```diff
@@ -0,0 +1,19 @@
+const { expect } = require('chai');
+const tshirts = require('./tshirts');
+
+describe('size function', () => {
+    it('should return an error for invalid input', () => {
```

Comment:
Nice addition of test :)
It gives good guidance to implement

Reply:
Thank you :)